readthedocs.yaml

```
@@ -0,0 +1,18 @@
+version: 2
+
+build:
+  image: testing
+
+python:
+  version: 3.9
+  install:
+    - requirements: requirements.txt
```

I just wanted to chime in here to say that this config file has been improved a little and you can use "Ubuntu 20.04" Docker image instead of `testing` here. See the docs at https://docs.readthedocs.io/en/stable/config-file/v2.html#build

```suggestion
build:
  os: ubuntu-20.04
  tools: 
    python: "3.9"

python:
  install:
    - requirements: requirements.txt
```

---

Yeah I saw, this needs a full rebase too. Thanks for pointing it out though!

A